index.html

```diff
@@ -194,13 +194,13 @@ <h1 class="mb-4 text-xl font-bold print:hidden">
     </div>
 
     <div
-      class="container mx-auto w-[210mm] border px-[9mm] py-[13.5mm] print:w-full print:max-w-full print:border-0"
+      class="container mx-auto w-[210mm] border px-[9mm] py-[12.5mm] print:w-full print:max-w-full print:border-0"
```

Comment:
For my Brother HL-L8240CDM I had to change this to 14.5, so it seems there is no generic solution :-/